samples/net/sockets/civetweb/src/libc_extensions.c

```diff
@@ -0,0 +1,84 @@
+#include <zephyr.h>
+#include <string.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "libc_extensions.h"
+
+#define FN_MISSING() printf("[IMPLEMENTATION MISSING : %s]\n", __func__)
+
+size_t strcspn(const char *s1, const char *s2)
+{
+	int i, j;
+
+	for (i = 0; i < strlen(s2); ++i) {
+		for (j = 0; j < strlen(s1); ++j) {
+			if (s1[j] == s2[i]) {
+				return j;
+			}
+		}
+	}
+
+	return strlen(s1);
+}
+
+int iscntrl(int c)
+{
+	/* All the characters placed before the space on the ASCII table
+	 * and the 0x7F character (DEL) are control characters.
+	 */
+	return (int)(c < ' ' || c == 0x7F);
+}
+
+struct tm * gmtime(const time_t *ptime)
+{
+	FN_MISSING();
+
+	return NULL;
+}
+
+size_t strftime(char *dst, size_t dst_size,
+		const char *fmt,
+		const struct tm *tm)
+{
+	FN_MISSING();
+
+	return 0;
+}
+
+double difftime (time_t end, time_t beg)
+{
+	return end - beg;
+}
+
+static char* generic_strerr = "[STRERR NOT INMPLEMENTED]";
+
+char *strerror(int err)
+{
+	return generic_strerr;
+}
+
+int sscanf(const char * s, const char * format, ...)
+{
+	FN_MISSING();
+
+	return 0;
+}
+
+double atof(const char* str)
+{
+	/* XXX good enough for civetweb uses */
+	return (double)atoi(str);
+}
+
+long long int strtoll(const char* str, char** endptr, int base)
+{
+	FN_MISSING();
```

Comment:
will strtoul be enough for civetweb? It is implemented

Reply:
It should, right!